test/actions/user_actions_test.js

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai'
+
+import * as actionCreators from '../../web/static/js/actions/user'
+
+describe('addUser', () => {
+  it('should create an action to add user to users list', () => {
+    const user = { given_name: 'Tiny Rick' }
+    const expectedAction = { type: 'ADD_USER', user }
+
+    expect(actionCreators.addUser(user)).to.deep.equal(expectedAction)
```

Comment:
same comment as below. try to stay away from pulling expectations out into variables, as it's easier to read if they're inline